src/Vm.sol

```diff
@@ -128,7 +128,7 @@ interface VmSafe {
     function load(address target, bytes32 slot) external view returns (bytes32 data);
 
     // Signs data
-    function sign(string memory privateKey, bytes32 digest) external pure returns (bytes memory sig);
+    function sign(string calldata privateKey, bytes32 digest) external pure returns (bytes calldata sig);
```

Comment:
Oh sorry, i missed the second "calldata" in return type.
Ok, you are correct, it is better to set it to memory for the code readability
If the sign() would real solidity function you would be 100% correct. Since it is part of cheatcodes I dont think it matters, but yeah, better to change it.